drivers/can/can_net.c

```diff
@@ -12,13 +12,39 @@ LOG_MODULE_REGISTER(net_can, CONFIG_CAN_NET_LOG_LEVEL);
 
```

Comment:
revise the function names in this commit. Public functions should always have net_<domain>_ prefix (so here: net_can_).

And it is nicer to have can_ as prefix for local function. For instance is_for_translator() -> can_is_pkt_for_translator()

Apply that everywhere

Reply:
Done.